server/remote/meeting.go

```diff
@@ -3,17 +3,21 @@
 
 package remote
 
-import "time"
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/serializer"
+)
 
 type FindMeetingTimesParameters struct {
-	ReturnSuggestionReasons   *bool               `json:"returnSuggestionReasons,omitempty"`
-	LocationConstraint        *LocationConstraint `json:"locationConstraint,omitempty"`
-	TimeConstraint            *TimeConstraint     `json:"timeConstraint,omitempty"`
-	MeetingDuration           *time.Duration      `json:"meetingDuration,omitempty"`
-	MaxCandidates             *int                `json:"maxCandidates,omitempty"`
-	IsOrganizerOptional       *bool               `json:"isOrganizerOptional,omitempty"`
-	MinimumAttendeePercentage *float64            `json:"minimumAttendeePercentage,omitempty"`
-	Attendees                 []Attendee          `json:"attendees,omitempty"`
+	ReturnSuggestionReasons   *bool                 `json:"returnSuggestionReasons,omitempty"`
+	LocationConstraint        *LocationConstraint   `json:"locationConstraint,omitempty"`
+	TimeConstraint            *TimeConstraint       `json:"timeConstraint,omitempty"`
+	MeetingDuration           *time.Duration        `json:"meetingDuration,omitempty"`
+	MaxCandidates             *int                  `json:"maxCandidates,omitempty"`
+	IsOrganizerOptional       *bool                 `json:"isOrganizerOptional,omitempty"`
+	MinimumAttendeePercentage *float64              `json:"minimumAttendeePercentage,omitempty"`
+	Attendees                 []serializer.Attendee `json:"attendees,omitempty"`
```

Comment:
should we use a pointer here?

Reply:
no copied the same from the intial code.